utils/proxy/rc_mocked_responses_asm_activate_only.json

```diff
@@ -10,6 +10,6 @@
         "raw": "eyJhc20iOnsiZW5hYmxlZCI6dHJ1ZX19"
       }
     ],
-    "targets": "eyJzaWduZWQiOnsiZXhwaXJlcyI6IjIwMjItMDktMTdUMTI6NDk6MTVaIiwic3BlY192ZXJzaW9uIjoiMS4wLjAiLCJjdXN0b20iOnsib3BhcXVlX2JhY2tlbmRfc3RhdGUiOiJhYWFhYSJ9LCJ0YXJnZXRzIjp7ImRhdGFkb2cvMi9BU01fRkVBVFVSRVMvYXNtX2ZlYXR1cmVzX2FjdGl2YXRpb24vY29uZmlnIjp7ImN1c3RvbSI6eyJ2IjoxfSwiaGFzaGVzIjp7InNoYTI1NiI6IjE1OTY1OGFiODViZTcyMDc3NjFhNDExMTE3MmIwMTU1ODM5NGJmYzc0YTFmZTFkMzE0ZjIwMjNmN2M2NTZkYiJ9LCJsZW5ndGgiOjI0fX0sInZlcnNpb24iOjIzMzM3MzkzfSwic2lnbmF0dXJlcyI6W3sia2V5aWQiOiJURVNUX0tFWV9JRCIsInNpZyI6ImRhODhmZTg3NzVjNjM4MzFiNGU3Nzk2MWIyYTk5NzhkYjM2YTNmZGUzMWExZmJjMGMyMTA2ZWRmMWQ4ZDJkYjI2NzAyN2U4NTI0YzI2NTg5MDhkMzA1Yjc3YTQ5NmE5ZjFkMWM1ZWQ2MjFiNmMwZjA2MTA4OTk5Y2FlOGFmOTBlIn1dfQo="
```

Comment:
The encoded targets here is missing the field `signed` - `_type` which according to the documentation is mandatory and its value has to be targets. This new encoded strings is the same as the previous one but with the that field added

Reply:
Just for the record. Before:

```json
{
  "signed": {
    "expires": "2022-09-17T12:49:15Z",
    "spec_version": "1.0.0",
    "custom": {
      "opaque_backend_state": "aaaaa"
    },
    "targets": {
      "datadog/2/ASM_FEATURES/asm_features_activation/config": {
        "custom": {
          "v": 1
        },
        "hashes": {
          "sha256": "159658ab85be7207761a4111172b01558394bfc74a1fe1d314f2023f7c656db"
        },
        "length": 24
      }
    },
    "version": 23337393
  },
  "signatures": [
    {
      "keyid": "TEST_KEY_ID",
      "sig": "da88fe8775c63831b4e77961b2a9978db36a3fde31a1fbc0c2106edf1d8d2db267027e8524c2658908d305b77a496a9f1d1c5ed621b6c0f06108999cae8af90e"
    }
  ]
}
```

After:
```json
{
  "signed": {
    "_type": "targets",
    "expires": "2022-09-17T12:49:15Z",
    "spec_version": "1.0.0",
    "custom": {
      "opaque_backend_state": "aaaaa"
    },
    "targets": {
      "datadog/2/ASM_FEATURES/asm_features_activation/config": {
        "custom": {
          "v": 1
        },
        "hashes": {
          "sha256": "159658ab85be7207761a4111172b01558394bfc74a1fe1d314f2023f7c656db"
        },
        "length": 24
      }
    },
    "version": 23337393
  },
  "signatures": [
    {
      "keyid": "TEST_KEY_ID",
      "sig": "da88fe8775c63831b4e77961b2a9978db36a3fde31a1fbc0c2106edf1d8d2db267027e8524c2658908d305b77a496a9f1d1c5ed621b6c0f06108999cae8af90e"
    }
  ]
}
```